crates/sui-graphql-rpc/src/context_data/sui_sdk_data_provider.rs

```diff
@@ -461,14 +461,13 @@ fn convert_bal(b: sui_json_rpc_types::Balance) -> Balance {
     }
 }
 
-pub(crate) fn convert_to_epoch(
+pub(crate) fn _convert_to_epoch(
```

Comment:
if no longer used, we can just remove

Reply:
Let's remove it all together (when we remove the sui_sdk_data_provider)